cypress/fixtures/company-settings/approval-process.json

```diff
@@ -0,0 +1,4 @@
+{
+  "approvalProcessFor": "qa",
+  "approver": "Group"
```

Comment:
Maybe `approvedBy` is a better naming. What do you think?

Reply:
Maybe you can use this file name: `approval-form.json`

And later below usage like this:

```approvalForm.peopleOrGroup
approvalForm.approvedBy```

Reply:
I agree with that.